protostar/commands/declare/declare_command.py

```diff
@@ -0,0 +1,101 @@
+from logging import Logger
+from pathlib import Path
+from typing import List, Optional
+
+from protostar.cli.command import Command
+from protostar.commands.deploy.deploy_command import DeployCommand
+from protostar.deployer import Deployer
+from protostar.deployer.deployer import InvalidNetworkConfigurationException
+from protostar.deployer.gateway_response import SuccessfulDeclareResponse
+from protostar.protostar_exception import ProtostarException
+
+
+class DeclareCommand(Command):
+    def __init__(self, deployer: Deployer, logger: Logger):
+        self._deployer = deployer
+        self._logger = logger
+
+    @property
+    def name(self) -> str:
+        return "declare"
+
+    @property
+    def description(self) -> str:
+        return "Sends a declare transaction to StarkNet."
+
+    @property
+    def example(self) -> Optional[str]:
+        return None
+
+    @property
+    def arguments(self) -> List[Command.Argument]:
+        return [
+            Command.Argument(
+                name="contract",
+                description=("The path to the compiled contract."),
```

Comment:
```suggestion
                description=("A path to a compiled contract."),
```

Reply:
@MaksymilianDemitraszek is the `A` article necessary here?

Reply:
NestJS doesn't use articles e.g.
"Path to webpack configuration."
"The name of the generated component."

https://docs.nestjs.com/cli/usages

---

> If it is required to be concise, e.g. in [headlines](https://en.wikipedia.org/wiki/Headline), signs, labels, and notes, articles are often omitted along with certain other [function words](https://en.wikipedia.org/wiki/Function_word)
https://en.wikipedia.org/wiki/English_articles